src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/ref/Microsoft.Extensions.DependencyInjection.Abstractions.cs

```diff
@@ -9,11 +9,11 @@ namespace Microsoft.Extensions.DependencyInjection
     public static partial class ActivatorUtilities
     {
         public static Microsoft.Extensions.DependencyInjection.ObjectFactory CreateFactory([System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors)] System.Type instanceType, System.Type[] argumentTypes) { throw null; }
-        public static Microsoft.Extensions.DependencyInjection.ObjectFactory<T> CreateFactory<[System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors)] T>(System.Type[] argumentTypes) { throw null; }
+        public static Microsoft.Extensions.DependencyInjection.ObjectFactory<T> CreateFactory<T>(System.Type[] argumentTypes) { throw null; }
```

Comment:
I believe you'll need to add back the `DynamicallyAccessedMembersAttribute` here and elsewhere. The API gen tools may have removed it?

Reply:
If I fix it manually, it doesn't compile because it says that compatibility is broken. When I build with `dotnet build /p:ApiCompatValidateAssemblies=false` and then with `dotnet msbuild /t:GenerateReferenceAssemblySource` it does remove these attribute.

Not sure what I am doing wrong here

Reply:
Yeah I had to rewrite the file manually